3DCalendar.xcodeproj/project.pbxproj

```diff
@@ -441,9 +449,9 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = Y32EQUAEWX;
+				DEVELOPMENT_TEAM = 7UMFPW78PV;
```

Comment:
Before we open source this we'll want to remove this. This can be done by choosing your target and selecting "none" as the team